### ISSUE TITLE
[BUG] Impossible to restart Microshift instance

### ISSUE THREAD

**Issue by slemeur** [crc-org/crc-extension#195]
I have made the update of OpenSHift Local from 2.28 to 2.30.
<img width="1645" alt="Screenshot 2023-12-15 at 13 32 29" src="https://github.com/crc-org/crc-extension/assets/1636769/dc8c3230-6d67-4ec9-9a34-98e8b28f84c0">

Then I tried to restart my microshift environnement (which I created the day before)
![Screenshot 2023-12-15 at 13 34 28](https://github.com/crc-org/crc-extension/assets/1636769/10ef6c9d-69e3-4cbc-8282-05410ecf470f)

### General information

  * OS: macOS
  * Podman Desktop 1.7.0-next

## CRC version
```bash
# Put `crc version` output here
```
  
## CRC status
```bash
# Put `crc status --log-level debug` output here
```

## CRC config
```bash
# Put `crc config view` output here
```

## Host Operating System
```bash
# Put the output of `cat /etc/os-release` in case of Linux
# put the output of `sw_vers` in case of Mac
# Put the output of `systeminfo` in case of Windows
```

### Steps to reproduce

  1. 
  2. 
  3. 
  4. 

### Logs

Before gather the logs try following if that fix your issue
```bash
$ crc delete -f
$ crc cleanup
$ crc setup
$ crc start --log-level debug
```

Please consider posting the output of `crc start --log-level debug`  on http://gist.github.com/ and post the link in the issue.

**Comment by slemeur** [comment]
It's also impossible after deleting the previous instance to do create a new one

**Comment by slemeur** [comment]
From the CLI
`$ crc start
Is 'crc daemon' running? Cannot reach daemon API: Get "http://unix/api/version": dial unix /Users/slemeur/.crc/crc-http.sock: connect: connection refused`

**Comment by praveenkumar** [comment]
From the cli can you try `$ crc setup` and then start?

**Comment by slemeur** [comment]
yes that's what I ended up doing - but the user will never understand that

**Comment by slemeur** [comment]
is there anyway, we can handle that for the user?

**Comment by praveenkumar** [comment]
Have to check with extension side but when `Cannot reach daemon API: Get "http://unix/api/version"` means `crc setup` required.

**Comment by gbraad** [comment]
@jeffmaury @dgolovin FYI. i do not think this will happen often, but needs to be resolved.

**Comment by slemeur** [comment]
>Cannot reach daemon API: Get "http://unix/api/version" means crc setup required.

Then why not adding it in the error message? As a user I like error messages, but I prefer how to work around them :)
